setup.py

```diff
@@ -30,12 +34,16 @@ def get_version():
     'pytest-asyncio>=0.5.0',
     'pytest-cov>=2.3.1',
     'logbook>=1.0.0',
+    'flake8>=3.0.4',
+    'isort>=4.2.5',
+    'collective.checkdocs>=0.2',
+    'Pygments>=2.1.3'  # required by checkdocs
```

Comment:
Just as good as I can understand your point, I know that this will be dependency hell if everyone else does it the same way. I don't want that to happen. `>=` on every package will sometimes break things, yes, but `==` will always break things if more than one package requires the same dependency.